CalibFormats/SiPixelObjects/interface/PixelCalibBase.h

```diff
@@ -28,7 +28,7 @@ namespace pos{
     PixelCalibBase();
     virtual ~PixelCalibBase();
     virtual std::string mode() const {return mode_;}
-    virtual void writeXMLHeader(  pos::PixelConfigKey &key, 
+    virtual void writeXMLHeader(  pos::PixelConfigKey key, 
```

Comment:
This is not an override and changes the semantics of the call (i.e. key is not changed but copied).